Stratis.Bitcoin/Features/BlockStore/BlockStoreLoop.cs

```diff
@@ -133,19 +139,35 @@ public sealed class BlockStoreLoop
             StartLoop();
         }
 
+        /// <summary>
+        /// Adds a block to Pending Storage
+        /// <para>
+        /// The BlockStoreSignaler calls AddToPending.
+        /// </para>
+        /// <para>
+        /// Only add the block to pending storage if:
+        ///     1: The block does exist on the chain
+        ///     2: The store's tip is less than the block to add's height
+        /// </para>
+        /// </summary>
+        /// <param name="block"></param>
```

Comment:
XMLDOC again - empty comment - again something as simple as `/// <param name="block">Block to add.</param>` fixes it.